components/studio/test/test.ps1

```diff
@@ -24,7 +24,7 @@ Copy-Item "$(hab pkg path core/7zip)/bin/*" "bin/7zip"
 Copy-Item "$(hab pkg path core/hab-plan-build-ps1)/bin/*" "bin/"
 
 try {
-    & bin/hab-studio.bat new
+    & bin/powershell/pwsh.exe -NoProfile -ExecutionPolicy bypass -NoLogo -File "bin/hab-studio.ps1" new
```

Comment:
What's the context in which this test file is run? Is there any way to share this with the same invocation that encoded in the Rust code now?

At the very least, it would be good to add a comment here referencing the Rust code (and vice versa) to ensure that future changes are made in both places.

Reply:
this is a "unit" test at least in the sense that the test can run directly on the studio code without requiring a compiled hab cli. This basically just replaces what was in the .bat file and is a pretty boilerplate/ubiquitous way to call into a powershell script in an embedded manner.